pom.xml

```diff
@@ -235,7 +235,25 @@
                     </execution>
                 </executions>
             </plugin>
-
+            <plugin>
+               <groupId>org.jacoco</groupId>
+               <artifactId>jacoco-maven-plugin</artifactId>
+               <version>0.8.5</version>
```

Comment:
Can you update the jococo plugin to latest version? 
It seems the latest one is 0.8.11
https://mvnrepository.com/artifact/org.jacoco/jacoco-maven-plugin/0.8.11

Reply:
@m-tilab 
the file (target/site/jacoco/jacoco.xml) will be create after build execution and the version of plugin update to latest version thanks for recommendation.